src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/AccountLegalEntityController.cs

```diff
@@ -12,6 +12,7 @@ namespace SFA.DAS.CommitmentsV2.Api.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
+    [Authorize]
```

Comment:
The attribute can be removed from AccountLegalEntityController's actions now too.